geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -155,7 +167,7 @@
                         <phase>package</phase>
                         <configuration>
                             <attach>true</attach>
-                            <finalName>geomesa-accumulo-${project.version}</finalName>
+                            <finalName>geomesa-accumulo_${scala.binary.version}-${project.version}</finalName>
```

Comment:
so when are we using the version parameter vs hard-coding it? it seems like the other references to our geomesa projects are all hard-coded (so far as i've read)